### ISSUE TITLE
Needs Automation

### ISSUE THREAD

**Issue by eddt** [marketingincolor/mic-dev-website#1]
Would like to find a way to have the "main" page at our [DEV site](http://dev.marketingincolor.com) automatically create links to the physical structure of the server setup.

Otherwise, perhaps we could use PHP arrays to designate and automate the display of the links. @adamdoe any ideas here?

**Comment by adamdoe** [comment]
Hmmm,

I will look into this and get back to you @eddt.

**Comment by eddt** [comment]
Cool – no rush or anything – just something I’d like to eventually see with our DEV setup!

**Comment by adamdoe** [comment]
@eddt comments on the latest commit?

**Comment by eddt** [comment]
I’m liking the way that you pieced this together here, @adamdoe. It’s really easy to READ/EDIT when it comes to making changes and adjustments. The only thing I think still needs a little work is the UI here, to get the layout a little “sharper” overall.

**Comment by adamdoe** [comment]
@eddt any suggestions on the UI?

**Comment by eddt** [comment]
As we discussed, a way to get some more info beyond the title/url link might make for better presentation on the page. And visually, if it looked sort of like a spreadsheet with column titles… Other than that the only “ui” comment I had was that I don’t think it needs a “footer” component (the bottom blue bar).

Feel free to experiment visually here and try different things to present the info!

**Comment by adamdoe** [comment]
@eddt I'm not sure if you get emails on the commits to this issue. I've updated the repo and pushed it live if you would like to see the changes.

**Comment by eddt** [comment]
Yeah, don’t know why I’m not getting notified on Commit for these – will take a closer look in a few, but at first glance I like the coding you’ve got. And just wanted to remind you that we DO have our OWN CDN for FontAwesome now! Just check out the OneNote -> MIC Development -> Resources -> FontAwesome CDN – don’t be afraid to use it!

**Comment by adamdoe** [comment]
Added our CDN
